tgui/packages/tgui/interfaces/AdminPDA.js

```diff
@@ -0,0 +1,122 @@
+import { useBackend } from '../backend';
+import { Section, Dropdown, Input, Box, TextArea } from '../components';
+import { Button, ButtonCheckbox } from '../components/Button';
+import { Window } from '../layouts';
+
+let windowSize;
+
+let user;
+let spam = false;
+
+let name, job;
+
+let messageText;
```

Comment:
You can not have these variables to store the state, as they're completely disconnected from React lifecycle and do not re-render the application on change. React has a bunch of its own conventions for managing state, so you need to get used to it a bit.

You must use `useLocalState` function (less refactoring), or https://reactjs.org/docs/state-and-lifecycle.html.